Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -500,7 +500,7 @@ static void logProcessPoolState(const WebProcessPool& pool)
 
 #if HAVE(VIDEO_RESTRICTED_DECODING)
 #if PLATFORM(MAC)
-    if (!isFullWebBrowser() || isRunningTest(WebCore::applicationBundleIdentifier())) {
+    if (!isFullWebBrowser()) {
```

Comment:
I think it would be good to remove this check now that isFullWebBrowser() is returning the expected value for Safari on macOS.

Reply:
Sorry I don't get it: which check should we remove? `if (!isFullWebBrowser()) {` ? Then we will create `trustdExtensionHandle` unconditionally?